.github/workflows/build_and_run_test.yml

```diff
@@ -37,23 +33,41 @@ jobs:
         tmpdir=$(mktemp -d /tmp/tmp.XXXXXX)
         echo "::set-output name=tmpdir::$tmpdir"
 
+    - name: Start docker containers
+      run: |
+        cd docker/generated
+        docker-compose up -d --build
+
     - name: Run integration test
       run: |
         tmpdir=${{ steps.set-tmpdir.outputs.tmpdir }}
         ts-tests/scripts/run-test.sh $tmpdir
       shell: bash {0}
 
-    - uses: actions/upload-artifact@v2
+    - name: Archive logs if test fails
+      uses: actions/upload-artifact@v2
       if: ${{ failure() }}
       with:
         name: litentry-parachain-debug.log
         path: ${{ steps.set-tmpdir.outputs.tmpdir }}/**/*
         retention-days: 7
 
+    - name: Push docker image if test passes
+      if: ${{ success() }}
+      run: |
+        [ -z "$(docker images --filter=dangling=true -q)" ] || docker rmi -f $(docker images --filter=dangling=true -q)
+        docker images
+        docker push -a litentry/litentry-parachain
+
     - name: Clean up
       if: ${{ always() }}
       run: |
         tmpdir=${{ steps.set-tmpdir.outputs.tmpdir }}
-        ts-tests/scripts/clean-up.sh $tmpdir
+        docker images
```

Comment:
I'd like to put these commands in a `cleaup.sh` script.

Reply:
> I'd like to put these commands in a `cleaup.sh` script.

I can do that